cli/src/command/generate_template.rs

```diff
@@ -22,10 +23,13 @@ use std::path::PathBuf;
 
 use crate::{
     command::{ExonumCommand, StandardResult},
-    config::{CommonConfigTemplate, GeneralConfig},
+    config::{GeneralConfig, NodePublicConfig},
     io::save_config_file,
 };
 
+/// Default supervisor service mode
+pub const DEFAULT_SUPERVISOR_MODE: SupervisorMode = SupervisorMode::Simple;
```

Comment:
Why don't implement `Default` trait for `SupervisorMode` ?

Reply:
It was implemented, and I requested it to be removed. I'll quote myself:

>I don't like it _(remark: it == `impl Default for Mode`)_. Simple mode is not default by its nature, it's default within exonum-cli and nothing more than it. On the Supervisor level those modes do have equal rights.